src/MatrixClientPeg.js

```diff
@@ -123,10 +126,11 @@ class MatrixClient {
             try {
                 localStorage.clear();
             } catch (e) {
-                console.warn("Error using local storage");
+                console.warn("Error clearing local storage", e);
             }
         }
         this.guestAccess.markAsGuest(Boolean(isGuest));
+        // ...not to be confused with Matrix.createClient()...
         createClient(hs_url, is_url, user_id, access_token, this.guestAccess);
```

Comment:
Given `createClient` on React SDK isn't called in many places, I would be tempted to rename it to avoid all confusion (and then the comments are obsolete!).